tests/test_index_order.py

```diff
@@ -32,6 +32,7 @@
     "dell_powerconnect",
     "enterasys",
     "extreme",
+    "extreme_exos",
```

Comment:
Not too opinionated, just want to call out this is the new os type name.

Reply:
Yes, hopefully this is in line with [https://ktbyers.github.io/netmiko/docs/netheremiko/extreme/extreme_exos.html](https://ktbyers.github.io/netmiko/docs/netheremiko/extreme/extreme_exos.html) and in accordance with your guidelines for this codebase.